app/jobs/send_expired_vacancy_feedback_email_job.rb

```diff
@@ -4,7 +4,10 @@ class SendExpiredVacancyFeedbackEmailJob < ApplicationJob
   def perform
     expired_vacancies.group_by(&:publisher_user).each do |user, users_vacancies|
       FeedbackPromptMailer.prompt_for_feedback(user.email, users_vacancies).deliver_later unless user.email.nil?
-      Rails.logger.info("Sidekiq: Sending feedback prompt emails for #{users_vacancies.count} vacancies") unless user.email.nil?
+
+      unless user.email.nil?
```

Comment:
You can move line 6 into this block and remove the additional unless at the end of that line, minor refactoring.

Reply:
Resolved in 38041b7